packages/nuxeo-platform-explorer-package/src/main/resources/package.xml

```diff
@@ -9,6 +9,7 @@
   <home-page>http://doc.nuxeo.com/x/VoVH</home-page>
   <platforms>
     <platform>server-@NUXEO_TARGET_VERSION@</platform>
+    <platform>server-2021.*</platform>
```

Comment:
should have been lts-2021 --> will issue a maintenance release for this (20.3.1)